gpt_computer_agent/utils/db.py

```
@@ -2,14 +2,23 @@
 import uuid
 from dotenv import load_dotenv
 
+try:
+    from .folder import currently_dir, artifacts_dir, media_dir
+    from .kot_db import kot_db_
+    from .user_id import *
```

Ruff (0.8.2)

`from .user_id import *` used; unable to detect undefined names

(F403)